commands/bash/container/add-aws-assume-role-config/add-aws-assume-role-config

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env bash
+set -eu -o pipefail
+
+function add_aws_assume_role_config {
```

Comment:
It seems this function uses a mix of parameters and env variables to operate, how are we choosing which go where?
My only concern is that reusing functions that do this kind of mix can get hairy

Reply:
BASTION_ACCOUNT_PROFILE and BASTION_ACCOUNT_ENABLED will always be constant within an invocation of aladdin. All parameters may be different in a single invocation of aladdin (e.g. publish role/publish_profile/publish_mfa_enabled could be distinct from aws_role/aws_profile/aws_mfa_enabled - see the _handle_aws_config function) duration is also a parameter in case someone wants to call it from bash to re-up their creds. 

AWS_DEFAULT_REGION should also never change